nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
[OPTIONAL] I suggest you add a message that should be thrown to the user for a better experience😉.
it can look like this👍

```suggestion
    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```

Reply:
Message has been added. Thank you @Ibizugbe